llvm/include/llvm/Analysis/TargetTransformInfoImpl.h

```diff
@@ -74,6 +74,10 @@ class TargetTransformInfoImplBase {
   unsigned getInliningCostBenefitAnalysisProfitableMultiplier() const {
     return 8;
   }
+  int getInliningLastCallToStaticBonus() const {
+    // This is same as InlineConstants::LastCallToStaticBonus.
+    return 15000;
```

Comment:
Just use LastCallToStaticBonus instead of hoping it stays the same?

Reply:
I don't want to include `llvm/Analysis/InlineCost.h` here because here it is a header as well, but apparently forward declaration doesn't work because the `InlineConstants::LastCallToStaticBonus` defined in a header as well. Any suggestions?